packaging/windows/msi/mk_msi.debug

```diff
@@ -4,7 +4,7 @@ set -eu
 # Sign exe files
 #
 mkdir -p "$SRC_ROOT/$BUILD_DIR/signed"
-CERT_FILE="$SRC_ROOT/$BUILD_DIR/$CERT_FILE_NAME"
+CERT_FILE="$SRC_ROOT/$BUILD_DIR/cert/$CERT_FILE_NAME"
```

Comment:
Since `CERT_FILE_NAME` is no longer passed by the Makefile, doesn't this need to be changed?

Reply:
it is used by debug build.

Reply:
Right it's executed here https://github.com/logdna/logdna-agent-v2/blob/c002e675027809c1a1203fa77db18b536e1d1d25/Makefile#L538

But with your changes there's no longer a `$CERT_FILE_NAME`. So doesn't it need to be changed to reference the debug cert file `mezmo_cert_debug.pfx`

Reply:
debug build is local - built by developer. developer is using debug make target for that.

Reply:
Resolved, the `CERT_FILE_NAME` is given a default value in an environment file https://github.com/logdna/logdna-agent-v2/blob/d64e541aff3a61cc81c39226ef13ef665a6a17be/packaging/windows/msi/mk_env#L15